examples/mnit_moles/src/moles.nit

```diff
@@ -90,38 +107,114 @@ class Hole
 			ey > y - dy and ey < y
 	end
 
-	fun hit
+	# Draw this hole and content to `display`
+	fun draw(display: Display)
 	do
-		if hitted then return
-
-		if trap then
-			up = false
-			game.points -= 5
-			if game.points < 0 then game.points = 0
-		else
-			hitted = true
-			game.points += 1
+		# The hole itself
+		var img = app.assets.empty
+		var dx = 300.0*display_scale
+		var dy = 256.0*display_scale
+		img.scale = display_scale
+		display.blit(img, x-dx.to_i+display_offset_x, y-dy.to_i+display_offset_y)
+
+		# The mole in the hole, or other content
+		var content = self.content
+		if content != null then
+			content.draw(display, x, y)
 		end
 	end
 end
 
+# Content of a `Hole`
+class HoleContent
+	# Image
+	var img: Image
+
+	# Offset of the horizontal center of the hole
+	var dx: Float
+
+	# Offset of the vertical center of the hole
+	var dy: Float
+
+	# Hit this hole content
+	fun hit(game: Game, hole: Hole, event: PointerEvent) do end
+
+	# Draw this content to `display`
+	fun draw(display: Display, x, y: Int)
+	do
+		img.scale = display_scale
+		display.blit(img,
+			x-dx.to_i+display_offset_x,
```

Comment:
Sorry, I thought that the goal of reviews was to ensure the quality and the readability of the source code.

As said in http://nitlanguage.org/manual/syntax.html#Style, the use of spaces around binary operators is encouraged. I don't think that enforcing the respect of our own rules is a waste of time.

So sorry to waste our time. It won't happen again.